src/objects/RecurringDonationChangeHistory__c.object

```diff
@@ -288,45 +300,57 @@
         <type>Currency</type>
     </fields>
     <fields>
-        <fullName>Previous_Campaign__c</fullName>
+        <fullName>PreviousCampaign__c</fullName>
         <deleteConstraint>SetNull</deleteConstraint>
         <description>The previous Campaign for the related Recurring Donation.</description>
         <externalId>false</externalId>
         <inlineHelpText>The previous Campaign for the related Recurring Donation.</inlineHelpText>
         <label>Previous Campaign</label>
         <referenceTo>Campaign</referenceTo>
         <relationshipLabel>Recurring Donation Change History (Previous Campaign)</relationshipLabel>
-        <relationshipName>RD_Change_History_Previous_Campaign</relationshipName>
+        <relationshipName>RDChangeHistoryPreviousCampaign</relationshipName>
         <required>false</required>
         <trackTrending>false</trackTrending>
         <type>Lookup</type>
     </fields>
     <fields>
-        <fullName>Previous_Payment_Method__c</fullName>
-        <description>The previous Payment Method for the related Recurring Donation.</description>
+        <fullName>PreviousInstallmentFrequency__c</fullName>
+        <description>The previous Installment Frequency for the related Recurring Donation.</description>
         <externalId>false</externalId>
-        <inlineHelpText>The previous Payment Method for the related Recurring Donation.</inlineHelpText>
-        <label>Previous Payment Method</label>
+        <inlineHelpText>The previous Installment Frequency for the related Recurring Donation.</inlineHelpText>
+        <label>Previous Installment Frequency</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
```

Comment:
This is a numeric field in the RecurringDonation and RecurringDonationSchedule objects.

Reply:
Good catch, that's what I get for adding a new field late on a Friday 😬